pkg/kubelet/kuberuntime/helpers_linux.go

```diff
@@ -54,11 +52,8 @@ func milliCPUToQuota(milliCPU int64) (quota int64, period int64) {
 		return
 	}
```

Comment:
@szuecs Don't we have to use old behavior here when the alpha feature flag is not set?

```
	if !utilfeature.DefaultFeatureGate.Enabled(kubefeatures.CPUCFSQuotaPeriod) {
		period = quotaPeriod
	}
```

Reply:
I added additional unit tests to make sure we have the right values if feature gate is enabled or disabled